openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+while [ -n "$1" ]; do
+	case "$1" in
+		--version|-v) export VERSION=1; break;;
+		--url) export BASE_URL="$2"; shift;;
+		--uuid) export UUID="$2"; shift;;
+		--key) export KEY="$2"; shift;;
+		--verify_ssl) export verify_ssl="$2"; shift;;
+		--interval) export INTERVAL="$2"; shift;;
+		--monitored_interfaces) export MONITORED_INTERFACES=$2; shift;;
+		-*)
+			echo "Invalid option: $1"
+			exit 1
+		;;
+		*) break;
+	esac
+	shift;
+done
+
+if [ "$VERSION" -eq "1" ]; then
+	VERSION=$(cat /etc/monitoring/VERSION)
```

Comment:
make sure that anything related to this package stays in `/etc/openwisp/monitoring`, there's already an `/etc/openwisp` directory created by openwisp, but make sure it's created if it doesn't exist, just to be extra sure code won't fail.